app/models/drive_backup.rb

```diff
@@ -13,6 +14,10 @@ def backup
     end
   end
 
+  def available?
+    config.service_account_json.present?
```

Comment:
The name comes from Google terminology. A service account is a special kind of Google API setup which is specifically intended to connect from server to server, with no user in the middle. A regular API key would require a user in the middle to authorize the access. The JSON part comes from how it is set up... when you set up the service account, you get a JSON file with all the configuration. This environment variable holds the contents of the service account JSON file you get when you set up the service account.